contact.md

```diff
@@ -48,7 +48,7 @@ If you do not have a technical issue, the fastest way to get help is to start in
 | Vipps through a partner (for merchants) | Please contact your partner. |
 | Vipps through a POS (Point of Sale) vendor | Please contact your POS vendor. |
 | Vipps through a PSP (Payment Service Provider) | Please contact your PSP. |
-| Partner assistance (for partners) | See: [Vipps Partners](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner) and the [Partner API](https://vippsas.github.io/vipps-developer-docs/docs/APIs/partner-api). |
+| Partner assistance (for partners) | Please contact your partner manager or contact us by email: [partnerbestilling@vipps.no](mailto:partnerbestilling@vipps.no). Check out the [Partner API](https://vippsas.github.io/vipps-developer-docs/docs/APIs/partner-api). |
```

Comment:
I intentionally linked to the partner page because I think some partners will find the info they need on their own, and also because it’s good to make sure partners know about the partner pages. Showing the email address on the contact us page can mean that [#team-partnerbestillling](https://vippsas.slack.com/archives/CMBF3EXB6) get more email than they need to.

Reply:
I fix